addons/account/models/account_move.py

```diff
@@ -2416,6 +2416,10 @@ class AccountMoveLine(models.Model):
         currency_field='company_currency_id',
         compute='_compute_balance',
         help="Technical field holding the debit - credit in order to open meaningful graph views from reports")
+    cumulated_balance = fields.Monetary(string='Cumulated Balance', store=False,
+        currency_field='company_currency_id',
+        compute='_compute_cumulated_balance',
+        help="Technical field holding the debit - credit in order to open meaningful graph views from reports")
```

Comment:
Graph views? Shouldn't we simply say it's used in the general ledger view?

Reply:
shhhht, nobody saw the copy-paste from above :no_mouth: